GCRCatalogs/catalog_configs/proto-dc2_v2.1.yaml

```diff
@@ -0,0 +1,14 @@
+subclass_name: AlphaQGalaxyCatalog
```

Comment:
@dkorytov please change this line to `subclass_name: alphaq.AlphaQGalaxyCatalog`